selenium_tests/class_roster/page_objects/class_roster_main_page.py

```diff
@@ -0,0 +1,34 @@
+"""
+This file models the Class Roster Main (Landing) Page
+"""
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.common.by import By
+from selenium_tests.class_roster.page_objects.class_roster_base_page_object \
+    import ClassRosterBasePageObject
+
+
+class Locators(object):
+    CLASS_ROSTER_BREADCRUMB = (By.XPATH, './/h1[contains(., "Class Roster")]')
```

Comment:
@elliottyates, hmmm, I seemed to be stuck on how to fix this. Unlike the other tests which have a exact match on "manage course", this is part of the breadcrumbs it's not an exact match because the full breadcrumb nav also includes "manage course > "  Tried against <h1> and <nav> and tried using the console, but I haven't been successful yet.   I'll have to dig a bit more on this one, but may have to pick your brains on it next week.